<launch>

    <node name="lar_vision_marker_detector_1" output="screen" pkg="lar_vision" type="lar_vision_marker_detector">

        <param name="camera_topic" value="/asus1/rgb/image_raw"/>
        <param name="camera_name" value="asus1"/>
        <param name="marker_id" value="600"/>
        <param name="no_view" value="1"/>
        <param name="marker_size" value="0.1"/>
        <param name="hz" value="5"/>
        <param name="camera_info_file_name" value="/home/lar/Desktop/workspace_parma/src/lar_vision/data/calibrations/asus1.yaml"/>

    </node>
    

</launch>
